src/modules/mentors/mentors.controller.ts

```diff
@@ -206,7 +206,12 @@ export class MentorsController {
     };
 
     const res: any = await this.mentorsService.updateApplication(applicationDto);
-    await this.emailService.send<SendDataRejectParams>(emailData);
+    try {
+      await this.emailService.send<SendDataRejectParams>(emailData);
+      await this.emailService.addMentor(user);
+    } catch (error) {
+      console.log(error);
```

Comment:
How about using `console.error` here instead?

```suggestion
      console.error(error);
```